### PR TITLE
Navbar/features

### PR DESCRIPTION
**Pull request by jacquesme**
Hi Oleg, could you please review my code?

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -45,10 +45,13 @@ module.exports = {
         xs: '9px',
         sm: '10px',
         md: '18px',
+        'md-40': '40px',
         xl: '200px',
         'xl1': '240px',
         xxl: '300px',
-        xxxl: '380px'
```
Comment (oleg-vysotsky) [inline tailwind.config.js:51]: Please, replace xxl, xxxl etc. with 2xl, 3xl.

File: tailwind.config.js [hunk 2]
```diff
@@ -45,10 +45,13 @@ module.exports = {
         xs: '9px',
         sm: '10px',
         md: '18px',
+        'md-40': '40px',
         xl: '200px',
         'xl1': '240px',
```
Comment (oleg-vysotsky) [inline tailwind.config.js]: You really don't need it. It's bad that your size will be a little different, better to organize it universally...

h-xs = h-2 (8px)
h-xl ~= h-48 (192px)
h-xl1 ~= h-64 (256px)
...

File: components/PopUp.vue [hunk 3]
```diff
@@ -0,0 +1,68 @@
+<template>
+
+  <div class="popup">
+    <div v-for="(item, key) in items" :key="key">
+      <div class="popup-item"><a href="#"><span class="popup-item_svg" v-html="item.svg"></span> {{ item.name }}</a></div>
+    </div>
+      <a href="#" class="popup-logOff"><p class="popup-logOff_item">{{ logOff.name }}<span class="popup-logOff_item_svg" v-html="logOff.svg"></span></p></a>
```
Comment (oleg-vysotsky) [inline components/PopUp.vue]: Code style. Don't use the camelcase here:

Just `popup-log-off`

File: components/Menu.vue [hunk 4]
```diff
@@ -1,68 +1,147 @@
...
+        .close {
+          position: relative;
           display: inline-block;
-          fill: #747474;
-          height: 1rem;
-          width: 1rem;
+          width: 50px;
+          height: 50px;
+          overflow: hidden;
+          &::before, &::after {
```
Comment (oleg-vysotsky) [inline components/Menu.vue]: You don't need :: here, use just `:before`

Comment (oleg-vysotsky) [reply components/Menu.vue]: And in all other pseudo-classes

**Comment by jacquesme** [comment]
Hi Oleg, 
Please check my update

**Pull request merged**